src/pages/releases.md

```diff
@@ -0,0 +1,175 @@
+[mysql2]: https://www.npmjs.com/package/mysql2
+
+## Microsoft SQL Server (mssql) Support Table
+
+MSSQL requires the use of the [tedious] npm package.  
+[Read more about this here](/docs/v7/other-topics/dialect-specific-things/#microsoft-sql-server-mssql).
+
+| Sequelize   | [SQL Server][mssql] | [tedious] |
+|-------------|---------------------|-----------|
+| 7 (alpha)   | 2017, 2019          | ^14.4.0   |
+| 6 (current) | 2012-2019           | ^8.3.0    |
+
+[mssql]: https://endoflife.date/mssqlserver
+[tedious]: https://www.npmjs.com/package/tedious
+
+## SQLite Support Table
+
+Sequelize uses the `sqlite3` npm library.  
+[Read more about this here](/docs/v7/other-topics/dialect-specific-things/#sqlite).
+
+:::caution
+
+sqlite3 has not been released since february 2021, and has security vulnerabilities and installation issues on newer platforms.
+
+We recommended using the [@vscode/sqlite3](https://github.com/microsoft/vscode-node-sqlite3) fork 
+by replacing the version with `npm:@vscode/sqlite3@5.0.7` like in the example below.
+
+It is supported in both v6 & v7.
+
+```json
+{
+  "dependencies": {
+    "sqlite3": "npm:@vscode/sqlite3@5.0.7"
+  }
+}
+```
+
+:::
+
+| Sequelize   | [sqlite3]                                                      |
+|-------------|----------------------------------------------------------------|
+| 7 (alpha)   | `npm:@vscode/sqlite3@^5.0.7`                                   |
+| 6 (current) | `npm:@vscode/sqlite3@^4.0.12`, or `npm:@vscode/sqlite3@^5.0.7` |
+
+[sqlite3]: https://www.npmjs.com/package/@vscode/sqlite3
+
+## Snowflake Support Table
+
+Snowflake requires the use of the [snowflake-sdk] npm package.  
+[Read more about this here](/docs/v7/other-topics/dialect-specific-things/#snowflake).
+
+:::note
+
+While this dialect is included in Sequelize,
+support for Snowflake is limited as it is not handled by the core team.
+
+:::
+
+| Sequelize   | [Snowflake](https://www.snowflake.com/pricing/) | [snowflake-sdk] |
+|-------------|-------------------------------------------------|-----------------|
+| 7 (alpha)   | all                                             | ^1.6.0          |
+| 6 (current) | all                                             | ^1.6.0          |
+
+[snowflake-sdk]: https://www.npmjs.com/package/snowflake-sdk
+
+## Db2 Support Table
+
+Db2 requires the use of the [ibm_db] npm package.  
+[Read more about this here](/docs/v7/other-topics/dialect-specific-things/#db2).
+
+:::note
+
+While this dialect is included in Sequelize,
+support for Db2 is limited as it is not handled by the core team.
+
+:::
+
+| Sequelize   | [Db2][db2] | [ibm_db] |
+|-------------|------------|----------|
+| 7 (alpha)   | >= 11.5    | ^2.8.0   |
+| 6 (current) | >= 11.5    | ^2.8.0   |
+
+[db2]: https://www.ibm.com/support/pages/db2-distributed-end-support-eos-dates
+[ibm_db]: https://www.npmjs.com/package/ibm_db
+
+## Db2 for IBM i Support Table
+
+*Db2 for IBM i* requires the use of the [odbc] npm package.  
```

Comment:
Why is this italic?

Reply:
Initially because it's long and I wanted to emphasize that this was the whole name of the dialect, but it's not very useful and I can remove it for consistency

Reply:
Italic removed